manifests/e/EugeneZrazhevsky/WeblocOpener/2.0.2/EugeneZrazhevsky.WeblocOpener.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.10.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
+
+PackageIdentifier: EugeneZrazhevsky.WeblocOpener
+PackageVersion: 2.0.2
+PackageLocale: en-US
+Publisher: benchdoos
+PublisherUrl: https://github.com/benchdoos
+PublisherSupportUrl: https://github.com/benchdoos/WeblocOpener/issues
+Author: Eugene Zrazhevsky
+PackageName: WeblocOpener
+PackageUrl: https://github.com/benchdoos/WeblocOpener
+License: MIT
+LicenseUrl: https://github.com/benchdoos/WeblocOpener/blob/master/LICENSE
+Copyright: Copyright © 2016-2023 Eugene Zrazhevsky
+ShortDescription: Open, create, and edit Apple .webloc files on Windows.
+Description: WeblocOpener is a simple way to open, create and edit Apple .webloc files on Windows. It can convert between .webloc, .url and .desktop shortcuts, and .webarchive to .html.
+Moniker: weblocopener
+Tags:
+- webloc
+- url
+- shortcut
+- converter
+- apple
+- html
+ReleaseNotes: Minor improvements and bug fixes.

```

release notes are available
```suggestion
ReleaseNotes: |-
  What’s new
  Improvements
  - Improved the stability of the application and some of its components
  - Reduced the size of the application and its components
  - Added a setting to not convert .webarchive and open the original site if it's available
  Fixed
  - The installer has been fixed, now it supports newer versions of Java
  - Fixed context menu for DuckDuckGo and Brave
  - Fixed file opening with specific symbols
```